laravel-app/app/Http/Controllers/PostController.php

```diff
@@ -18,9 +18,6 @@ public function __construct(PostService $postService)
         $this->postService = $postService;
     }
 
-    /*
-    * View all instances of posts in home page.
-    */
```

Comment:
Why remove the comments on the method? For every method you create, there must be a comment at least.

Reply:
Yes sir, but according to sir raf I should remove comments to methods that are self explanatory to avoid over documentation. Should I include comments for every methods sir Mark?

Reply:
Added comments per method sir. Thank you